python/lsst/ts/mtaos/mtaos.py

```diff
@@ -519,8 +519,8 @@ async def do_runWEP(self, data):
                     self.execution_times["RUN_WEP"] = []
                 self.execution_times["RUN_WEP"].append(time.time() - start_time)
 
-            self.model.query_ocps_results(
-                "LSSTComCam/quickLook", intra_visit_id, extra_visit_id
+            await self.model.query_ocps_results(
+                "u/saluser/ra_wep_testing4", intra_visit_id, extra_visit_id
```

Comment:
I don't like this being hard-coded at all, but perhaps there's no better mechanism existing, and now's not the time for inventing one, but I do wonder if there's something better possible. If we bump that (and we will, I'm certain), having to redeploy MTAOS to do so is pretty bad.